.github/workflows/build-deploy-docs.yml

```diff
@@ -1,4 +1,6 @@
+---
 name: "DOC - Build and deploy"
+permissions: read-all
```

Comment:
Adding `permissions: read-all` to the top level of the GH Actions workflow files resolves the `CKV2_GHA_1` checkov check ("Ensure top-level permissions are not set to write-all") that was failing:

https://github.com/tylervz/npm-groovy-lint/actions/runs/4949858452/jobs/8852638177#step:4:225